src/components/Bridge/TransferPanel/TransferPanel.tsx

```diff
@@ -95,6 +95,7 @@ const TransferPanel = ({ className, setShowAssetSelector }: Props) => {
         (errors) => ({ ...errors, gas: undefined }),
       );
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
i'm not sure this is a good decision/design choice - ideally we should rework the implementation so that there are no warnings?